doc/introduction/introducing_zephyr.rst

```diff
@@ -83,23 +99,10 @@ small-footprint OSes:
    * *Power Management Services* such as tickless idle and an advanced idling
      infrastructure.
 
-   * *Native Networking Stack* fully featured and optimized, including
-     LwM2M and BSD sockets compatible support. OpenThread support is also
-     provided - a mesh network designed to securely and reliably
-     connect hundreds of products around the home.
-
-   * *Bluetooth Low Energy* 5.0 support including a Bluetooth
-     qualification-ready BLE controller, and BLE Mesh.
-
-   * *File system* with Newtron Flash Filesystem (NFFS) and FAT
+   * *File system* with Newtron Flash Filesystem (NFFS) and FATFS
```

Comment:
please also add "FCB (Flash Circular Buffer) for memory constrained applications"

Reply:
addressed both, thanks.